packages/entity-server/package.json

```diff
@@ -12,14 +12,14 @@
     "url": "git+https://github.com/beyondessential/tupaia"
   },
   "author": "Beyond Essential Systems <admin@tupaia.org> (https://beyondessential.com.au)",
-  "main": "dist/index.js",
+  "main": "dist/main.js",
```

Comment:
As far as I can tell specifying the main file here doesn't seem to make a difference... might back this change out...